rust/docker-compose.yaml

```diff
@@ -1,17 +1,12 @@
-version: "3.4"
```

Comment:
I was getting an error because version 3.0 does not support the cache_from property, I'm not sure if it will work now

Reply:
3.0 is old but 3 will use the latest version in the major release of "3.X"

Reply:
yay okey